test/game_test.rb

```diff
@@ -7,62 +7,63 @@
 
 class GameTest < Minitest::Test
   def setup
-    @cruiser = Ship.new("Cruiser", 3)
-    @submarine = Ship.new("Submarine", 2)
-    @player1 = Board.new
-    @player2 = Board.new
-    @game = Game.new(@player1, @player2)
+    # @cruiser = Ship.new("Cruiser", 3)
+    # @submarine = Ship.new("Submarine", 2)
+    # @player1 = Board.new
+    # @player2 = Board.new
+    @game = Game.new
   end
 
   def test_it_exists
     assert_instance_of Game, @game
   end
 
   def test_it_has_attributes
-    assert_equal @player1, @game.player
-    assert_equal @player2, @game.computer
+    assert_instance_of Board, @game.player
+    assert_instance_of Board, @game.computer
```

Comment:
This gets the name, but should we just assert_instance to all? Dunno if it would be convention but hey, it works!